fetch-restart.sh

```
@@ -0,0 +1,114 @@
+    echo ""
+    echo "=========================================="
+    echo "✅ Fetch-Restart Complete!"
+    echo "=========================================="
+    echo "📡 Backend:  http://${DISPLAY_HOST}:${BACKEND_PORT}"
+    echo "🎨 Frontend: http://${DISPLAY_HOST}:${FRONTEND_PORT}"
+    echo "📊 Logs:     /workspace/logs/supervisor.log"
+    echo "=========================================="
+else
+    # Local: kill by port, then start
+    echo "🧹 Cleaning up existing processes..."
+    lsof -ti:$BACKEND_PORT 2>/dev/null | xargs kill -9 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)